prometheus/indico_prometheus/metrics.py

```diff
@@ -0,0 +1,149 @@
+except ImportError:
+    LIVESYNC_AVAILABLE = False
+
+
+num_active_events = metrics.Gauge('indico_num_active_events', 'Number of Active Events')
+num_events = metrics.Gauge('indico_num_events', 'Number of Events')
+
+num_active_users = metrics.Gauge('indico_num_active_users', 'Number of Active Users (logged in in the last 24h)')
+num_users = metrics.Gauge('indico_num_users', 'Number of Users')
+
+num_categories = metrics.Gauge('indico_num_categories', 'Number of Categories')
+
+num_active_attachment_files = metrics.Gauge('indico_num_active_attachment_files', 'Number of attachment files')
+num_attachment_files = metrics.Gauge(
+    'indico_num_attachment_files',
+    'Total number of attachment files, including older versions / deleted'
+)
+
+size_active_attachment_files = metrics.Gauge(
+    'indico_size_active_attachment_files',
+    'Total size of all active attachment files (bytes)'
+)
+size_attachment_files = metrics.Gauge(
+    'indico_size_attachment_files',
+    'Total size of all attachment files, including older versions / deleted (bytes)'
+)
+
+num_notes = metrics.Gauge('indico_num_notes', 'Number of notes')
+
+num_active_rooms = metrics.Gauge('indico_num_active_rooms', 'Number of active rooms')
+num_rooms = metrics.Gauge('indico_num_rooms', 'Number of rooms')
+
+num_bookings = metrics.Gauge('indico_num_bookings', 'Number of bookings')
+num_valid_bookings = metrics.Gauge('indico_num_valid_bookings', 'Number of valid bookings')
+num_pending_bookings = metrics.Gauge('indico_num_pending_bookings', 'Number of pending bookings')
+
+num_occurrences = metrics.Gauge('indico_num_booking_occurrences', 'Number of occurrences')
+num_valid_occurrences = metrics.Gauge('indico_num_valid_booking_occurrences', 'Number of valid occurrences')
+
+num_ongoing_occurrences = metrics.Gauge('indico_num_ongoing_booking_occurrences', 'Number of ongoing bookings')
+
+if LIVESYNC_AVAILABLE:
+    size_livesync_queues = metrics.Gauge('indico_size_livesync_queues', 'Items in Livesync queues')
+
+
+def update_metrics(active_user_hours):
+    """Update all metrics."""
+    now = now_utc()
+    num_events.set(Event.query.filter(~Event.is_deleted).count())
+    num_active_events.set(Event.query.filter(~Event.is_deleted, Event.start_dt <= now, Event.end_dt >= now).count())
+    num_users.set(User.query.filter(~User.is_deleted).count())
+    num_active_users.set(
+        User.query
+        .filter(Identity.last_login_dt > (now - timedelta(hours=active_user_hours)))
+        .join(Identity).group_by(User).count()
+    )
+    num_categories.set(Category.query.filter(~Category.is_deleted).count())
+
+    num_active_attachment_files.set(get_attachment_query().count())
+    num_attachment_files.set(
+        AttachmentFile
+        .query
+        .join(Attachment, AttachmentFile.attachment_id == Attachment.id)
+        .count()
+    )
+
+    attachment_subq = db.aliased(Attachment, get_attachment_query().subquery("attachment"))
```

Comment:
```suggestion
    attachment_subq = db.aliased(Attachment, get_attachment_query().subquery('attachment'))
```

(i need to finally run pyquotes to convert the last few double quotes in the plugin repo and then re-enable the flake8 rule for this ;))